### PR TITLE
Improve k8s configuration

### PR DESCRIPTION
**Pull request by luoluoyuyu**

### CODE REVIEW

**Comment by luoluoyuyu** [comment]
The results are as follows```helm install streampipes ./```
image
The results are as follows ```helm install streampipes ./ --set preferredBroker=kafka```
image

**Review by dominikriemer — APPROVED** [review]
Awesome!

**Review by tenthe — APPROVED** [review]

File: installer/k8s/README.md [hunk 1]
```diff
@@ -82,6 +82,160 @@ We retain the created persistent volume. You need to manually delete it:
...
+| streampipes.ingressroute.certResolverActive     | Flag for enabling certificate resolver for IngressRoute | true                                     |
+| streampipes.ingressroute.certResolver           | Certificate resolver for IngressRoute                   | ""                                       |
+
+
+###Extensions common parameters
+| Parameter Name                                  | Description                                             | Value                                    |
+|-------------------------------------------------|---------------------------------------------------------|------------------------------------------|
+| extensions.iiot.appName                         | IIoT extensions application name                        | extensions-all-iiot                      |
+| extensions.iiot.port                            | Port for the IIoT extensions application                | 8090                                     |
+| extensions.iiot.service.name                    | Name of the IIoT extensions service                     | extensions-all-iiot1                     |
```
Comment (RobertIndie) [inline installer/k8s/README.md:138]: Why is there a suffix 1 here? `extensions-all-iiot1`

File: installer/k8s/README.md [hunk 2]
```diff
@@ -82,6 +82,160 @@ We retain the created persistent volume. You need to manually delete it:
...
+| external.influxdb.port1                         | Port 1 for the InfluxDB service                          | 8083                                     |
+| external.influxdb.port2                         | Port 2 for the InfluxDB service                          | 8086                                     |
+| external.influxdb.port3                         | Port 3 for the InfluxDB service                          | 8090                                     |
+| external.influxdb.service.name                  | Name of the InfluxDB service                             | "influxdb"                               |
+| external.influxdb.service.port1                 | Port 1 for the InfluxDB service                          | 8083                                     |
+| external.influxdb.service.port2                 | Port 2 for the InfluxDB service                          | 8086                                     |
+| external.influxdb.service.port3                 | Port 3 for the InfluxDB service                          | 8090                                     |
```
Comment (RobertIndie) [inline installer/k8s/README.md:191]: It's better to give a more meaningful name for them.

**Comment by dominikriemer** [comment]
Oh sorry - I had the page still open and after merging I saw @RobertIndie 's comments appear - sorry for that!
@luoluoyuyu can you please address @RobertIndie comments in a new PR?

**Comment by RobertIndie** [comment]
@dominikriemer It doesn’t matter. :) It's okay that we address these comments in a new PR.

**Comment by luoluoyuyu** [comment]
hi @dominikriemer
I submitted a new pr to address the issue in the comments

**Pull request merged**